frontend/.env.local.example

```diff
@@ -0,0 +1,2 @@
+REACT_APP_POST_URL_PORT=":3000"
```

Comment:
i'd leave out the colon here

Reply:
Fixed [here](https://github.com/salsita/foosball-rating/pull/95/commits/0a92aa2b8d648b6df6e9301cec83602f84b58d28#diff-5696aa6f6ae27f14a99322ab211068ba)